src/drivers/pwm_out/PWMOut.cpp

```diff
@@ -587,6 +590,7 @@ void PWMOut::update_params()
 
 	if (_class_instance == CLASS_DEVICE_PRIMARY) {
 		prefix = "PWM_MAIN";
+		/// TODO: This should somehow be used as the prefix used for MixingOutput
```

Comment:
@dagar - this if/else stuff here was your work; how do you see it being combined with the MixingOutput modifications?